app/javascript/react/containers/SuperpowerShowContainer.js

```diff
@@ -6,15 +6,38 @@ import ReviewsContainer from './ReviewsContainer'
 class SuperpowerShowContainer extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {}
+    this.state = {
+      name: '',
+      description: '',
+      reviews: []
+    }
+  }
+
+  componentDidMount() {
+    let superpowerId = this.props.params.id
+    fetch(`/api/v1/superpowers/${superpowerId}`)
+    .then(response => response.json())
+    .then(json => {
+      this.setState({
+        name: json.superpower.name,
+        description: json.superpower.description,
+        reviews: json.reviews
+      })
+    })
   }
 
   render() {
     return(
       <div>Hello from superpower show page
-        <div> <SuperpowerDetailTile /></div>
-        <div> <ReviewsContainer /></div>
-      </div>
+        <SuperpowerDetailTile
+          name={this.state.name}
+          description={this.state.description}
+        />
+        <ReviewsContainer
+          reviews={this.state.reviews}
+        />
+        <ReviewFormContainer />
```

Comment:
The ReviewFormContainer shouldn't exist yet so please remove this!

Reply:
Actually, this brings up a really good question as this should make your website error out. Please, please tell me you're ALWAYS click-testing your site before submitting a PR? Make sure you actually boot it up and click around to make sure everything's loading up appropriately.